src/codegen/mod.rs

```diff
@@ -2062,6 +2077,13 @@ impl ToRustTy for Type {
                 let ident = ctx.rust_ident(&name);
                 quote_ty!(ctx.ext_cx(), $ident)
             }
+
+            TypeKind::ObjCInterface(..) => {
+                let name = item.canonical_name(ctx);
```

Comment:
clang will give an actual error if interface embedded in namespace. And to even get that error, it needs to be run with -x objective-c++ mode